resources/user-agents/browsers/opera-mobile/opera-mobile-generic.json

```diff
@@ -64,11 +102,10 @@
         {
           "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*)*Chrome/*Safari/* OPR/*",
           "platforms": [
-            "Android_1_0", "Android_1_1", "Android_1_5", "Android_1_6",
-            "Android_2_0", "Android_2_1", "Android_2_2", "Android_2_3",
-            "Android_4_0", "Android_4_1", "Android_4_2", "Android_4_3",
-            "Android_4_4", "Android_5_0", "Android_5_1",
-            "Android_7_1", "Android_7_0", "Android_6_1", "Android_6_0",
+            "Android_7_1", "Android_7_0",
```

Comment:
As mentioned in #1396, Opera for Android only supports Andriod 4.0+, so I removed these older versions of Android.